draft-ietf-git-using-github.md

```diff
@@ -188,10 +188,24 @@ IETF tools be able to determine this and easily find the policy and GitHub
 organization.
 
 A simple example of how to do this is to include a link to the GitHub
-organization on the WG Charter page in the datatracker under More Info.
-Similarly, if there are multiple mailing list options, links to those mailing
-lists should be given.  An example of this is at
-https://datatracker.ietf.org/wg/quic/charter/.
+organization on the WG Charter page in the datatracker.  Similarly, if there are
+multiple mailing list options, links to those mailing lists should be given.
+
+Repositories MUST include a copy or reference to the policy that applies to
+managing any documents they contain.  Updating the README or CONTRIBUTING file
+in the repository with details of the process ensures that the process is
+recorded in a stable location other than the mailing list archive.  This also
+makes Working Group policies available to casual contributors who might only
+interact with the GitHub repository.
+
+GitHub prominently links to the CONTRIBUTING file on certain pages.  This file
+SHOULD be used in preference to the README for information that new contributors
+need.  A link to the CONTRIBUTING file from the README is advised.
+
+In addition to Working Group policies, notices on repositories MUST include
+citations for BCPs 78 and 79 ({{!COPYRIGHT=RFC5378}}, {{!IPR=RFC8179}}) and
```

Comment:
Could we instead just mandate a reference to the note well? I think the IETF Code of Conduct and Anti-Harassment Procedures are at least as import as Copyrights and IPR, and singling out some parts of the Note Well doesn't make sense to me.

Reply:
I like this suggestion!

Reply:
I agree.  It also simplifies this text more, so double win.